docs/windows.md

```diff
@@ -371,17 +308,18 @@ $options = New-ScheduledJobOption -RunElevated
 Register-ScheduledJob -Name PrepareAntreaAgent -Trigger $trigger  -ScriptBlock { Invoke-Expression C:\k\antrea\Prepare-AntreaAgent.ps1 } -ScheduledJobOption $options
 ```
 
-#### 5. Run kubeadm to join the Node
+##### 5. Run kubeadm to join the Node
 
 On Windows Node, run the `kubeadm join` command to join the cluster. The token
 is provided by the control-plane Node.
 
```

Comment:
I feel better to merge the three paragraphs (line 313 to 320) to one.

Reply:
Merged into one.